Source/CarthageKit/Git.swift

```diff
@@ -22,13 +22,12 @@ public struct FetchCache {
 		lastFetchTimes.removeAll()
 	}
 
-	internal static func needsFetch(forURL url: GitURL) -> Bool {
-		guard let lastFetch = lastFetchTimes[url] else {
+    internal static func needsFetch(forURL url: GitURL) -> Bool {
```

Comment:
Carthage uses tabs instead of spaces for indentation -- could you please reformat your changes to do the same?